examples/TestDriver/ios/Podfile.lock

```diff
@@ -105,11 +105,11 @@ SPEC CHECKSUMS:
   DoubleConversion: bb338842f62ab1d708ceb63ec3d999f0f3d98ecd
   Folly: c89ac2d5c6ab169cd7397ef27485c44f35f742c7
   glog: e8acf0ebbf99759d3ff18c86c292a5898282dcde
-  React: 1fe0eb13d90b625d94c3b117c274dcfd2e760e11
-  react-native-heap: 381fd78115a2a1314dae2906b893221c9501d12d
-  RNGestureHandler: afde2addc517e85cc97a25bfe119acbee779a7ce
-  yoga: b1ce48b6cf950b98deae82838f5173ea7cf89e85
+  React: 01aa04500b2957c2767e1dff9fe12e09444a467c
+  react-native-heap: e75c689169e157e25a236be2c072bf24aa195824
+  RNGestureHandler: b65d391f4f570178d657b99a16ec99d09b8656b0
+  yoga: a5c0ba30ebe82c13612b4c9301ad29708b8978dc
 
 PODFILE CHECKSUM: 94963ca9da08a6f38a9019eb8da64ec0f890b712
 
-COCOAPODS: 1.5.3
+COCOAPODS: 1.6.1
```

Comment:
So this is one of those pain in the ass things, because you have a different version of Cocoapods than I do, clearly, and I didn't even realize there was an update available... We ran into this problem at Udacity, and it's just a small burden that dev teams take on, unless you start standardizing your ruby gems etc. which is a whole different PITA...